src/ucp/rndv/rndv.h

```diff
@@ -30,14 +35,8 @@ typedef struct {
     uint64_t          address;
     /* Size of the data for sending */
     size_t            size;
-    union {
-        /* RNDV proto flags, as defined by ucp_rndv_rts_flags_t */
-        uint8_t       flags;
-        /* RNDV operation status, used only by old version for TAG protocol,
-         * kept here for wire compatibility
-         */
-        ucs_status_t  status;
-    };
+    /* RNDV proto flags, as defined by ucp_rndv_rts_flags_t */
```

Comment:
can add `@ref ucp_rndv_rts_flags_t` to have a link in doxygen and keep the comment valid in case of type name changed